aries-site/src/examples/templates/app-layouts/anatomy/PageAnatomy.js

```diff
@@ -0,0 +1,24 @@
+import { useContext } from 'react';
+import { ThemeContext } from 'grommet';
+
+import { ContentArea } from './components';
+
+export const PageAnatomy = () => {
+  const theme = useContext(ThemeContext);
+  const height = theme.global.size.medium;
+  const width = `${(height.replace('px', '') * 4) / 3}px`;
+
+  return (
+    <ContentArea
+      title="Page Container"
+      border={{ color: 'border-weak' }}
+      gap="small"
+      round="xsmall"
+      height={height}
+      width={width}
+    >
+      <ContentArea title="Page Header" background="purple!" flex={false} />
```

Comment:
We had received feedback in the past that using these strong background colors was a bit too bold, which is why we went for the dashed borders on the current implementation -- wondering if we can find a middle ground